doc/dnf5_plugins/builddep.8.rst

```diff
@@ -0,0 +1,73 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _builddep_plugin_ref-label:
+
+#################
+ Builddep Command
+#################
+
+Synopsis
+========
+
+``dnf5 builddep [options] [<package>...]``
+
+
+Description
+===========
+
+Install missing dependencies for building an RPM package.
+
+.. warning:: Build dependencies in a source package (i.e. src.rpm) might be different
+             than you would expect because they were evaluated according macros
```

Comment:
I would replace "they" with "source packages". more clear

Reply:
It refers to dependencies, actually. But I've replaced the pronoun anyway.